Running nixpkgs-review, build seems to be failing on many kernel versions:
```
error: builder for '/nix/store/ya78kcd8c97i2g78p6z0xazhaiy1fslw-gasket-1.0-18-unstable-2023-09-05.drv' failed with exit code 2;
       last 10 log lines:
       > mkdir: cannot create directory '/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/build/.tmp_991': Permission denied
       > mkdir: cannot create directory '/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/build/.tmp_999': Permission denied
       > mkdir: cannot create directory '/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/build/.tmp_1016': Permission denied
       > mkdir: cannot create directory '/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/build/.tmp_1024': Permission denied
       > mkdir: cannot create directory '/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/build/.tmp_1050': Permission denied
       > /nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash: line 1: /nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/source/modules.order: Permission denied
       > make[3]: *** [/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/source/scripts/Makefile.build:462: /nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/source/modules.order] Error 1
       > make[2]: *** [/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/source/Makefile:1565: _module_/nix/store/m6bmrz149b72jpwr8r0skpld758safnn-linux-4.19.297-dev/lib/modules/4.19.297-hardened1/source] Error 2
       > make[1]: *** [Makefile:153: sub-make] Error 2
       > make: *** [Makefile:24: __sub-make] Error 2
```

--- a/pkgs/os-specific/linux/gasket/default.nix
+++ b/pkgs/os-specific/linux/gasket/default.nix
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     sha256 = "fcnqCBh04e+w8g079JyuyY2RPu34M+/X+Q8ObE+42i4=";
   };
 
-  makeFlags = [
+  makeFlags = kernel.makeFlags ++ [
     "-C"
     "${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
     "M=$(PWD)"
@@ -31,5 +31,6 @@ stdenv.mkDerivation rec {
     license = licenses.gpl2;
     maintainers = [ lib.maintainers.kylehendricks ];
     platforms = platforms.linux;
+    broken = versionOlder kernel.version "5.15";
   };
 }

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -27842,10 +27842,6 @@ with pkgs;
 
   fwts = callPackage ../os-specific/linux/fwts { };
 
-  gasket = callPackage ../os-specific/linux/gasket {
-    inherit (linuxPackages) kernel;
-  };
-
   gobi_loader = callPackage ../os-specific/linux/gobi_loader { };
 
   libossp_uuid = callPackage ../development/libraries/libossp-uuid { };

--- a/pkgs/top-level/linux-kernels.nix
+++ b/pkgs/top-level/linux-kernels.nix
@@ -345,6 +345,8 @@ in {
 
     fwts-efi-runtime = callPackage ../os-specific/linux/fwts/module.nix { };
 
+    gasket = callPackage ../os-specific/linux/gasket { };
+
     gcadapter-oc-kmod = callPackage ../os-specific/linux/gcadapter-oc-kmod { };
 
     hyperv-daemons = callPackage ../os-specific/linux/hyperv-daemons { };

